fractions/javaee/datasources/README.adoc

```diff
@@ -56,3 +56,66 @@ may be used when defining a datasource is as follows:
 |`prestodb`
 |===
 
+== Example datasource definitions
+
+=== MySQL
+
+This is an example of a MySQL datasource configuration with connection information, basic security, and validation options.
```

Comment:
From a stylistic standpoint, it would be better if the sentence started with just _An example...,_ and ended with a colon. However, it's not a hard requirement.

Reply:
Fixed.

Reply:
Thank you. :)